Test.C

```diff
@@ -388,6 +393,10 @@ int main(){
 
         // Save player score to leaderboard (prototype 2)
     }
+	
+	// Calling upon the function for adding/grabbing the score from the website
+	void addScore(char playerName, int score, int seed, int version);
```

Comment:
Lets just hardcode defaults for now.
```suggestion
	void addScore(player.name, player.currentCredits, 1, 0.01);
```